From 947b5ac34667ac4d1c295758e0347748b184b01d Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Fri, 9 Sep 2011 12:48:19 +0800
Subject: [PATCH 0424/2463] ENGR00156319-1 mxc edid: use vmode as aspect rate
 flag

Use vmode as aspect rate flag.
which is more easier to use.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc_edid.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_edid.h b/arch/arm/plat-mxc/include/mach/mxc_edid.h
index 7f76beb..59688f6 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_edid.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_edid.h
@@ -26,8 +26,8 @@
 #ifndef MXC_EDID_H
 #define MXC_EDID_H
 
-#define FB_MODE_ASPECT_4_3	0x10
-#define FB_MODE_ASPECT_16_9	0x20
+#define FB_VMODE_ASPECT_4_3	0x10
+#define FB_VMODE_ASPECT_16_9	0x20
 
 struct mxc_edid_cfg {
 	bool cea_underscan;
-- 
1.7.7.4

